### PR TITLE
Doc updates for Jaeger tracing port value

### PR DESCRIPTION
**Pull request by ljamen**
Doc for issue #5187 and #6094

### CODE REVIEW

**Comment by ljamen** [comment]
Hi @tjquinno  - There are a couple of ways to address this doc bug. I believe you were asking the team how to update. Did you get your answer?

Should the doc contain a brief note at least listing these and/or explaining workarounds if any? For example, propagation is no longer supported, but propagation: b3 is no longer needed because the Helidon's JaegerTracerBuilder class always includes the OpenTelemetry B3Propagator.

**Comment by tjquinno** [comment]
TBH I am not sure what workarounds, if any, are available for the settings that Jaeger no longer supports. Our config properties, then and now, simply mirror whatever Helidon's code can set on Jaeger's span exporter builder. Jaeger changed and so the settings we can expose changed as well.

Given that, it's probably clearer to _not_ list the specific settings we _no longer_ support (given that, at least at the moment, I don't know what, if any, workaround there might be). Maybe, just before the `include` of the generated config `.adoc` file, we could state something like this:

> Because Jaeger changed its client implementation some Jaeger settings exposed by earlier releases of Helidon are no longer available. Please note the currently-supported settings in the table below.

or something to that effect.

File: docs/config/io_helidon_tracing_jaeger_JaegerTracerBuilder.adoc [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ///////////////////////////////////////////////////////////////////////////////
 
-    Copyright (c) 2022 Oracle and/or its affiliates.
+    Copyright (c) 2023 Oracle and/or its affiliates.
```
Comment (tjquinno) [inline docs/config/io_helidon_tracing_jaeger_JaegerTracerBuilder.adoc:3]: Do not change this file (or any in the `docs/config` directory) manually.

They are all generated.

File: docs/includes/tracing/tracer-jaeger.adoc [hunk 2]
```diff
@@ -47,14 +47,16 @@ include::{rootdir}/includes/dependencies.adoc[]
```
Comment (tjquinno) [inline docs/includes/tracing/tracer-jaeger.adoc:47]: See my comment in the PR discussion for a suggested brief note to insert here.

File: docs/se/tracing.adoc [hunk 3]
```diff
@@ -203,9 +203,9 @@ tracing:
 routing.register(WebTracingConfig.create(config.get("tracing")));
 ----
 
-==== Path based configuration in Helidon Web Server
+==== Path-based configuration in Helidon Web Server
 
-For Web Server we have a path based support for configuring tracing, in addition
+For Web Server we have a path-based support for configuring tracing, in addition
```
Comment (tjquinno) [inline docs/se/tracing.adoc:208]: Very small. Maybe remove the "a" from before "path-based" here?

**Review by tjquinno — CHANGES_REQUESTED** [review]

**Comment by al3xandru** [comment]
This commit doesn't include a fix for https://helidon.io/docs/v3/#/se/guides/tracing (the snippet using private void getDefaultMessageHandler refers to and uese the io.opentracing instead of io.helidon.tracing API)

**Comment by ljamen** [comment]
Hi @al3xandru I'm working with Tim on that. This is still in draft mode until I've completed all of the updates. They will be included in this PR.

**Comment by ljamen** [comment]
Closing this PR and will include files in a new PR that does not include the generated tracing doc that was accidentally updated.

**Pull request closed without merging**